hazenlib/slice_width.py

```diff
@@ -362,7 +362,7 @@ def get_initial_trapezoid_fit_and_coefficients(profile, slice_thickness):
         n_ramp = 47
         n_plateau = 55
 
-    trapezoid_centre = round(np.median(np.argwhere(profile < np.mean(profile)))).astype(int)
+    trapezoid_centre = int(round(np.median(np.argwhere(profile < np.mean(profile)))))
```

Comment:
Why is this change necessary?

Reply:
This can be removed actually. Was an early change when I thought a numpy version change was causing problems.

Reply:
Resolved by 2ca01a3